packages/web-runtime/src/components/SidebarNav/SidebarNavItem.vue

```diff
@@ -83,12 +91,17 @@ export default {
   position: relative;
   align-items: center !important;
   display: flex !important;
-  justify-content: flex-start !important;
+  justify-content: space-between !important;
   padding: var(--oc-space-small) !important;
   border-radius: 5px;
   white-space: nowrap;
   user-select: none;
 
+  .oc-tag {
+    color: var(--oc-color-text-default);
+    background-color: var(--oc-color-background-highlight);
+    min-height: 24px !important;
```

Comment:
hm.. just seeing now that we lack a `size` property for the OcTag component. Would you mind adding it to the design system?

Reply:
for sure, but IDK which sizes we should use for s,m,l ...?

Reply:
We already have several components with a `size` prop. They all enforce the same size value naming scheme, all chose `medium` as default and apply sizings with predefined classes which all bring the same sizings. See for example here: https://github.com/owncloud/owncloud-design-system/blob/f99d9b35e7b8f68c5a259d6067b95b9e00460e46/src/components/atoms/OcButton/OcButton.vue#L181 and the location where the scss mixin is used.

Either you try with line height, or height. Whichever looks better.